weaviate/templates/weaviateStatefulset.yaml

```diff
@@ -24,18 +24,6 @@ spec:
       serviceAccountName: {{ .Values.backups.s3.serviceAccountName }}
       {{- end }}
       terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
```

Comment:
shouldn't be there `{{ .Values.affinity }}`?

Reply:
I just removed the first one.